src/Adapt/Adapt.stories.tsx

```diff
@@ -0,0 +1,66 @@
+// REACT
+import * as React from 'react';
+// STORYBOOK
+import { storiesOf } from '@storybook/react';
+// VENDOR
+import styled, { ThemeProvider } from '@xstyled/styled-components';
+// ANCHOR
+import { RootTheme } from '../../src/theme';
+import { Button } from '../Button';
+import { Typography } from '../Typography';
+import { Adapt } from './Adapt.component';
+import { object } from '@storybook/addon-knobs';
+
+const StyledStory = styled('div')`
+    padding: 2rem;
+    color: text.base;
+    font-family: base;
+`;
+
+storiesOf('Components/Adapt', module)
+    .add('Default', () => {
+        return (
+            <ThemeProvider theme={RootTheme}>
+                <StyledStory>
+                    <Adapt
+                        forEach={object('forEach', {
+                            xs: 'apple',
+                            md: 'strawberry',
+                            lg: 'blueberry',
+                        })}
+                    >
+                        {(value, breakpoint) => (
+                            <div>
```

Comment:
Going to want to add a `key` prop here, and for the other example. Getting fussed at in the console.